release/gff/gff.byn.gff_blin/source/welcome.htm

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Start Using GFF Amharic</title>
+  <style type="text/css">
+    p { font: 10pt Tahoma; }
+    h1 { font: bold 16pt Tahoma; color: #4444cc; margin-bottom: 2px }
+    h2 { font: bold 12pt Tahoma; color: #4444cc; }
+  </style>
+</head>
+<body>
+
+<h1>Start Using GFF Amharic</h1>
+
+<p>
+    This lexical model is intended for use with the Blin Keyman Keyboard on mobile devices, see:
+    <a href="https://keyman.com/blin/">https://keyman.com/blin/</a>

```

Comment:
Is this now?
```suggestion
    <a href="https://keyman.com/keyboards/gff_blin">https://keyman.com/keyboards/gff_blin</a>
```

Reply:
@dyacob Did you want to make this change? The original link doesn't work, so needs to be changed or removed.